esp/bindings/http/platform/httptransport.cpp

```diff
@@ -1565,11 +1565,10 @@ void CHttpRequest::parseCookieHeader(char* cookiestr)
         if(name.charAt(0) != '$')
         {
             cookie = new CEspCookie(name.str(), value.str());
+            cookie->setPath("/");
```

Comment:
I am not sure whether this will change anything. The Path has been set to '/' when new a CEspCookie (https://github.com/hpcc-systems/HPCC-Platform/blob/e28c39ef1b62911bae533b7dfd3a0d097b97de67/esp/platform/espsession.ipp#L57).